db/init.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE IF NOT EXISTS rha;
+
+CREATE TABLE events (
+    event_id BIGINT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255),
+    description TEXT,
+    start_time DATETIME,
+    event_type INT,
+    event_location_id BIGINT,
+    min_robins TINYINT,
+    max_robins TINYINT,
+    created_by VARCHAR(255),
```

Comment:
```suggestion
    created_by BIGINT UNSIGNED,
```

let's map this to user_id so that other user details can be referred if needed